pkg/endpointmanager/manager.go

```diff
@@ -479,16 +479,10 @@ func (mgr *EndpointManager) AddHostEndpoint(ctx context.Context, owner regenerat
 	epLabels := labels.Labels{}
 	epLabels.MergeLabels(labels.LabelHost)
 
-	if k8s.IsEnabled() {
```

Comment:
Are we sure we want to get rid of this check? I guess we don't run outside of k8s anyway.

Reply:
I don't think we need it. `node.GetLabels()` will return an uninitialized variable in that case; I have checked that it works fine by removing the `node.SetLabels(n.Labels)` statement below. The `Runtime*` tests are also passing so it should be okay.